package.json

```diff
@@ -29,7 +29,7 @@
     "@s-ui/lint": "2",
     "@s-ui/mono": "1",
     "@s-ui/precommit": "2",
-    "@s-ui/test": "1",
+    "@s-ui/test": "2",
```

Comment:
I made this change 4 days ago: https://github.com/SUI-Components/sui/commit/257d29d192ddf442e7365d35bc74736fd8ce8903